src/test/java/com/ms/zookeeper/TestZkUserAuthorizer.java

```diff
@@ -0,0 +1,47 @@
+package com.ms.zookeeper;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import com.ms.zookeeper.authz.ZkUserAuthorizer;
```

Comment:
com.ms - wrong namespace.

Reply:
I was wondering about that. Since it is in an ms repo. I thought it makes sense to use ms namespace. What would you prefer?